drivers/gnss/gnss_nmea_generic.c

```diff
@@ -148,24 +154,49 @@ static int gnss_nmea_generic_init(const struct device *dev)
 		return ret;
 	}
 
+#if CONFIG_PM_DEVICE
+	pm_device_init_suspended(dev);
+#else
 	ret = gnss_nmea_generic_resume(dev);
 	if (ret < 0) {
 		return ret;
 	}
```

Comment:
Would we consider this an API change?
This significantly changes the default driver behavior for a user already using `CONFIG_PM_DEVICE`. I think we need to add a note in the migration guide.